### DISCUSSION TITLE
GPUMD-v4.8

### DISCUSSION THREAD

**Discussion by brucefan1983** [brucefan1983/GPUMD#1325] [Announcements]
# GPUMD-v4.8

* Updates for the `nep` executable:
  * Changed to also fine-tune the descriptor parameters for the `fine_tune` keyword, which has been tested to lead to higher accuracy than fine-tuning the neural network parameters only. #1295
  * For qNEP models, added `lambda_q` and `lambda_z` to set the loss weights for the total charge (presently zero) and Born effective charge (BEC). #1293 

* New `gnep` executable:
  * This has been in existence for quite a few months, but only became mature recently. It provides an alternative way to train certain versions of NEP potential models. 

* Updates for the `gpumd` executable:
  * The `fix` keyword is in effective for the `minimize sd` keyword, which can be used to fix atoms during minimization. #1319 

<hr /><em>This discussion was created from the release <a href='https://github.com/brucefan1983/GPUMD/releases/tag/v4.8'>GPUMD-v4.8</a>.</em>
